src/lib.rs

```diff
@@ -89,6 +89,8 @@ pub struct Record {
     pub alarm: Option<Alarm>,
     /// The UAS location, which may be given in one of several declaration types.
     pub location: Location,
+    /// The UAS velocity given in ENU coordinates (relative to the UAS position and given in meters per second).
+    pub velocity: Option<CoordENU>,
```

Comment:
```suggestion
    #[serde(default)]
    pub velocity: Option<CoordENU>,
```
If velocity is not included in the input, ignore it. https://serde.rs/attr-default.html